src/BankConsole/Commands/CommandFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BankConsole.Commands
+{
+    class CommandFactory
+    {
+        private ICommand[] commands = {
+            new QuitCmd(),
+            new NewAccount(),
+            new SelectCmd(),
+            new DepositCmd(),
+            new Withdraw(),
+            new AuthorizeLoan(),
+            new ShowCmd(),
+            new AddInterest()
+        };
+
+        public ICommand GetCommand(int commandNumber)
+        {
+            return commands[commandNumber];
+        }
+
+        public string[] cmdNames()
+        {
+            List<string> names = new List<string>();
+            foreach( var cmd in commands)
+            {
+                names.Add(cmd.ToString());
+            }
+            return names.ToArray();
```

Comment:
use the LINQ h ere to convert command to names

Reply:
Ok, thaks for reviews